desc/magnetic_fields/_current_potential.py

```diff
@@ -744,6 +761,183 @@ def from_surface(
             check_orientation=False,
         )
 
+    def to_CoilSet(  # noqa: C901 - FIXME: simplify this
+        self,
+        num_coils,
+        step=1,
+        spline_method="cubic",
+        show_plots=False,
+        npts=128,
+        stell_sym=False,
+        figsize=(8, 6),
```

Comment:
might be better to just make a general `plot_kwargs` or something, to allow other kwargs to be specified